docs/customizing_dashboards.md

```diff
@@ -77,8 +77,8 @@ which are specified through the `.with_options` class method:
 
 **Field::BelongsTo**
 
-`:order` - Specifies the order of the dropdown menu, can be ordered by more
-than one column. e.g.: `"name, email DESC"`.
+`:order` - Specifies the order of either the dropdown menu and the table,
+can be ordered by more than one column. e.g.: `"name, email DESC"`.
```

Comment:
I think we can use a better phrasing here:

```suggestion
`:order` - Specifies the column used to order the records. It will apply both in
the table views and in the dropdown menu on the record forms. 
You can set multiple columns as well with direction. E.g.: `"name, email DESC"`.
```

Reply:
Done ✅